libvisual/tools/lv-tool/lv-tool.cpp

```diff
@@ -378,6 +379,11 @@ namespace {
       std::signal (SIGTERM, handle_termination_signal);
   }
 
+  const std::unordered_set<std::string> actors_to_skip {
```

Comment:
@hartwork, I just noticed that this is stuck in between function definitions. It should be brought to the top of the file.

Reply:
@kaixiong the intention was to be in proximity, can sure be argued. I'd like to pull the "if #219 was good enough, then this" card here, e.g. because the idea was to keep the two copies close. Does it matter enough to you to be fixed on both sides?

Reply:
@hartwork I would prefer it be fixed on both sides. Thanks in advance!

My strong preference is for globals, especially app settings, to be up top, rather than stuffed between function definitions. The centralisation makes them easier to find/view/tweak in one glance. This is also in keeping with the current organization of the file.

There are cases where proximity make more sense though, such as when the source file is sectioned into regions, one for each logical component. Other times, it would be extremely verbose or impossible to place a declaration at the beginning, like say for a lookup table, because that is initialized with a number of function calls, etc.

Reply:
> @hartwork I would prefer it be fixed on both sides. Thanks in advance!

@kaixiong alright, moved up :beers: